tmt/cli.py

```diff
@@ -541,6 +541,12 @@ def tests_lint(
     '-t', '--template', metavar='TEMPLATE',
     help=f'Test template ({_test_templates}).',
     prompt=f'Template ({_test_templates})')
+@option(
+    '--link', metavar='LINK',
+    help='Link to the relevant issue.')
+@option(
+    '--relation', metavar='<verifies|relates>',
+    help='Link relation to the relevant issue.')
```

Comment:
Use `type=click.Choices` for input validation and nicer metavar:

```python
@option(
    '--relation',
    type=click.Choice(['verifies', 'relates']),
    help='...')
```

`option()` will add metavar and validate the input to allow only one of the given strings.

Reply:
Thank you for your nice suggestion which simplifies the code a lot, @happz ! And the option `--reation` is updated, please review again, thanks!